src/runtime_src/core/include/xclbin.h

```diff
@@ -329,6 +329,20 @@ extern "C" {
         FAST_ADAPTER = 5
     };
 
+    // m_subtype
+    enum PS_SUBTYPE {
+        ST_PS = 0,
+        ST_DPU = 1,
+        ST_UNKNOWN = -1,
```

Comment:
There should not be an unknown enumeration type.  This enumeration represents possible bit values.   If an UNKNOWN value is used, then this value could never be used in the future.   In addition, assigning it to -1 leads to other issues for it assumes that this value is all 1s.  If in the future more bits are reserved, then this value becomes more difficult to support.
Please remove.

Reply:
removed